content/boilerplates/kubectl-multicluster-contexts.md

```diff
@@ -10,9 +12,11 @@
               cluster2   cluster2   user@foo.com   default
     {{< /text >}}
 
-* Export the following environment variables with the context names of your configuration:
+*   Store the context names of your clusters in environment variables:
 
     {{< text bash >}}
-    $ export CTX_CLUSTER1=<KUBECONFIG_CONTEXT_NAME_FOR_CLUSTER_1>
-    $ export CTX_CLUSTER2=<KUBECONFIG_CONTEXT_NAME_FOR_CLUSTER_2>
+    $ export CTX_CLUSTER1=$(kubectl config view -o jsonpath='{.contexts[0].name}')
```

Comment:
I like this, but we should probably make it clear that you can only do this if the two clusters you want to use are the first two contexts that you have. I think it might be a good idea to say (e.g., tip) something like `If you have more than two clusters in the context list and you want to configure your mesh using clusters other than the first two, you will need to manually set the environment variables to the appropriate context names.`

Reply:
@frankbu Thanks, I have added this tip in https://github.com/istio/istio.io/pull/4073/commits/ea3462be3c7840fcb5e14e7e77e2dfab004f2814.